### ISSUE TITLE
Rule convert es-dsl aggregation result is error

### ISSUE THREAD

**Issue by FXYGR** [Neo23x0/sigma#549]
I run commandline:
```
tools/sigmac -t es-dsl -c tools/config/winlogbeat.yml rules/windows/builtin/win_susp_failed_logons_single_source.yml
```
aggregation actual result:
```
"aggs": {
      "Workstation_count": {
        "terms": {
          "field": "Workstation.keyword"
        },
        "aggs": {
          "limit": {
            "bucket_selector": {
              "buckets_path": {
                "count": "Workstation_count"
              },
              "script": "params.count > 3"
            }
          }
        }
      }
    }
```
Expected results:
```
"aggs": {
    "winlog.event_data.WorkstationName_count": {
      "terms": { "field": "winlog.event_data.WorkstationName" },
      "aggs": {
        "agg": {
          "terms": {
            "field": "UserName"
          },
          "aggs": {
            "limit": {
              "bucket_selector": {
                "buckets_path": {
                  "count": "_count"
                },
                "script": "params.count > 3"
              }
            }
          }
        }
      }
    }
  }
```
In the link: https://github.com/Neo23x0/sigma/wiki/Specification
this is that:
```
Example: count(UserName) by SourceWorkstation > 3
This comparison counts distinct user names grouped by SourceWorkstations.
```
So please fix this bug, thanks.

**Comment by thomaspatzke** [comment]
I think this has been fixed by PR #526 - please verify.

**Comment by zouzias** [comment]
Hi @FXYGR, let us know if #526 fixes the issue for you as well.
